packages/anoncreds/src/protocols/proofs/v1/__tests__/v1-connectionless-proofs.e2e.test.ts

```diff
@@ -154,6 +155,18 @@ describe('V1 Proofs - Connectionless - Indy', () => {
       threadId: aliceProofExchangeRecord.threadId,
       state: ProofState.Done,
     })
+  }
+
+  test('Faber starts with connection-less proof requests to Alice', async () => {
+    await connectionlessTest()
+  })
+
+  test('Faber starts with connection-less proof requests to Alice return route set to false', async () => {
+    await connectionlessTest(false)
+  })
+
+  test('Faber starts with connection-less proof requests to Alice return route set to true', async () => {
+    await connectionlessTest(false)
   })
```

Comment:
We're not really testing whether the parameter is acutally used and influences the flow. It's the same test, ran three times, and it will behave the same as AFJ will already close the session. Maybe a unit tests is more in place here, or we check whether the sent message contains a `~transport` decorator with the correct value set?

Reply:
Added a check on the transport decortator to ensure that it has the correct value

Reply:
I only see it added in the v2 test. Can you also add it here?